### PR TITLE
Getting Started Updates for SIG-Windows: take 2

### PR DESCRIPTION
**Pull request by rjmorse**
This is a result of https://github.com/kubernetes/website/pull/7117 and issues with updating the source repo to resolve conflicts.

Commits have been cherry picked, and my author email updated to avoid a CLA org problem (delays for setting up new org)

Changes include:
- Remove the implicit recommendation to compile node binaries
- Clarify the TOR example configuration file
- Updates to sample yaml files and explain the OS tag usage of 1709
- Add to known issues list
- Remove mentioned required PR that has been merged
- Add next steps footer

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7535)

### CODE REVIEW

**Comment by rjmorse** [comment]
I finally got back to this, and the CLA looks fine this time around. Changes (only the 1 page) are staged at https://deploy-preview-7535--kubernetes-io-master-staging.netlify.com/docs/getting-started-guides/windows/

Zach would you mind taking a look when possible?
/assign @zacharysarah

File: docs/getting-started-guides/windows/index.md [hunk 1]
```diff
@@ -439,7 +440,13 @@ Some of these limitations will be addressed by the community in future releases
...
+- Hyper-V isolated containers are not supported. 
+- Windows container OS must match the Host OS. If it does not, the pod will get stuck in a crash loop.
+- Under the networking models of L3 or Host GW, Kubernetes Services are inaccessible to Windows nodes due to a Windows issue. This is not an issue if using OVN/OVS for networking.
+- Windows kubelet.exe may fail to start when running on Windows Server under VMWare Fusion [issue 57110](https://github.com/kubernetes/kubernetes/pull/57124)
+- Flannel and Weavenet are not yet supported
 
+## Next steps and resources
 
-> As of this writing, the Kube-proxy binary requires a pending Kubernetes [pull request](https://github.com/kubernetes/kubernetes/pull/56529) to work properly. You may need to [build](#build) the binaries manually to work around this. 
+- Support for Windows is in Beta as of v1.9 and your feedback is welcomed. For information on getting involved, please head to [SIG-Windows](https://github.com/kubernetes/community/blob/master/sig-windows/README.md)
```
Comment (michmike) [inline docs/getting-started-guides/windows/index.md]: nit, but maybe reword to "your feedback is welcome"

Comment (rjmorse) [reply docs/getting-started-guides/windows/index.md]: Sounds good. Updated

**Comment by bsteciuk** [comment]
@rjmorse this still needs to make it into the release-1.10 branch right?
